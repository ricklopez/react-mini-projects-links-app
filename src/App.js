import React from 'react';
import logo from './logo.svg';
import './App.css';
import Nav from './Nav';
import Actions from './Actions';

class App extends React.Component {
  constructor(props) {
    super(props);
    this.state = {date: new Date()};
  }
  
  render(){
    
    return (
      <div className="App">
        <div className="container">
        <div className="row">
          <div className="col">
             <Nav time={Date.now()}/>
             <Actions/>
                <div className="links-list  my-3">
                  <h4 className="display-4">My Links</h4>
                  <ul className="list-group">
                    <li className="list-group-item d-flex justify-content-between align-items-center link-item">
                      Cras justo odio
                      <span className="badge badge-primary badge-pill">14</span>
                    </li>
                    <li className="list-group-item d-flex justify-content-between align-items-center link-item">
                      Dapibus ac facilisis in
                      <span className="badge badge-primary badge-pill">2</span>
                    </li>
                    <li className="list-group-item d-flex justify-content-between align-items-center link-item">
                      Morbi leo risus
                      <span className="badge badge-primary badge-pill">1</span>
                    </li>
                  </ul>
                </div>
            </div>
          </div>
        </div>
      </div>
    );
    
  }
 
}

export default App;
