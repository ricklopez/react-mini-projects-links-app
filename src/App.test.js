import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import App from './App';

test('renders learn react Nav and List Item', () => {
  const component = renderer.create(<App />);
                                  
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
                               
});
