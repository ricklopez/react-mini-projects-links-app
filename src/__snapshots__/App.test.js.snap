// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react Nav and List Item 1`] = `
<div
  className="App"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <div
          className="nav my-3"
        >
          <ul
            className="nav mx-auto"
          >
            <li
              className="nav-item"
            >
              <a
                className="nav-link active"
                href="#"
              >
                Active
              </a>
            </li>
            <li
              className="nav-item"
            >
              <a
                className="nav-link"
                href="#"
              >
                Link
              </a>
            </li>
            <li
              className="nav-item"
            >
              <a
                className="nav-link"
                href="#"
              >
                Link
              </a>
            </li>
            <li
              className="nav-item"
            >
              <a
                className="nav-link disabled"
                href="#"
              >
                Disabled
              </a>
            </li>
          </ul>
        </div>
        <div
          className="actions-bar  my-3"
        >
          <button
            className="btn btn-primary btn-lg btn-block"
            onClick={[Function]}
            type="button"
          >
            Block level button
          </button>
        </div>
        <input
          onChange={[Function]}
        />
        <div
          className="links-list  my-3"
        >
          <h4
            className="display-4"
          >
            My Links
          </h4>
          <ul
            className="list-group"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
